src/components/ReactPortal.js

```diff
@@ -0,0 +1,21 @@
+import React, { useLayoutEffect, useState } from "react";
+import { createPortal } from "react-dom";
+
+const ReactPortal = ({root, children}) => {
```

Comment:
Hi @Emmanuel-Develops, first off, great job on these additions! They're freaking superb!!! 🔥 

Quick one though. Can we rename this function to `Portal` instead of `ReactPortal`?

Reply:
Definitely, I'll get on this right away

Reply:
When you're done with this. Kindly tag me 🙏🏼

Reply:
I'm done. @kaf-lamed-beyt